### DISCUSSION TITLE
Desktop Cruiser 4.0.0 import Garmin favorites

### DISCUSSION THREAD

**Discussion by GuG1050** [devemux86/cruiser#530] [Q&A] [answered]
After updating the desktop version from 3.40 to 4.0.0 importing of favorites (.gpx files) does not work anymore (deleting works). The import window appears and I can click the gpx-file. However the favorites list remains empty. Any change there or bug?

**Comment by devemux86** [comment]
@GuG1050 thanks for the report.

Can you post the gpx file to check it?

Is it a Garmin gpx?
There will be a fix in import of Garmin gpx. A Beta version to test is here:
- https://github.com/devemux86/cruiser/discussions/522#discussioncomment-9320399

**Comment by GuG1050** [comment, marked as answer]
Yes, its a Garmin gpx file exported from Basecamp. Import works with 4.0.1-b1 .
[20 Korsika_POI gpx.txt](https://github.com/devemux86/cruiser/files/15220978/20.Korsika_POI.gpx.txt)
